lib/Raven/Breadcrumbs/ErrorHandler.php

```diff
@@ -21,16 +21,11 @@ public function __construct(\Raven\Client $ravenClient)
 
     public function handleError($code, $message, $file = '', $line = 0, $context = array())
     {
-        $this->ravenClient->breadcrumbs->record(array(
-            'category' => 'error_reporting',
-            'message' => $message,
-            'level' => $this->ravenClient->translateSeverity($code),
-            'data' => array(
-                'code' => $code,
-                'line' => $line,
-                'file' => $file,
-            ),
-        ));
+        $this->ravenClient->leaveBreadcrumb($this->ravenClient->translateSeverity($code), Breadcrumb::TYPE_ERROR, 'error_reporting', $message, [
```

Comment:
Although having similar APIs between various SDK seems to be a good thing, and I agree that somehow it is, its also a limit to what can be done depending on the language itself. Given that every language has its own features, syntax, etc I don't see a big problem in having different APIs. Instead I would point more on having the same set of features across all SDKs (like the processors). By the way, can you be more clear about what concerns you have about this change?